e2e/scripts/check-nllb.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+DIR=/usr/local/bin
+. $DIR/common.sh
+
+function main() {
```

Comment:
I would add a comment to this saying that it's meant to be run after deleting the first controller node to ensure the cluster is still functional

Reply:
done